template/contracts/foundry.toml

```diff
@@ -6,9 +6,9 @@ fs_permissions = [{ access = "read", path = "./"}]
 solc="0.8.23"
 
 [rpc_endpoints]
-"${NETWORK}"="${RPC_URl}"
+base_sepolia="${RPC_URL}"
 
 [etherscan]
-"${NETWORK}"={key="${BLOCK_EXPLORER_API_KEY}"}
+base_sepolia={key="${BLOCK_EXPLORER_API_KEY}",url="${BLOCK_EXPLORER}"}
```

Comment:
It allows for any network and you can specify the network in `.env`

Reply:
Got it, but it's not a good practice to restrict the user to using only one chain/configuration at a time. If I want to deploy on multiple chains, I'm forced to switch the network in `.env` every time. The user should be allowed to add as many networks as they want, as per the Foundry book documentation: https://book.getfoundry.sh/reference/config/etherscan

Reply:
You'll also get a `bad fork_url provider: base_sepolia` when running for example `forge script script/BuyMeACoffee.s.sol --rpc-url $NETWORK`

Reply:
Yeah makes sense.

@gabrielstoica do you have a good example of `foundry.toml` you liked in the past that we can take a look.

Also if you are thinking an end-to-end experience from coding, to deploy and so on. How you would like to approach this?

And again, thank you for taking the time to share your thoughts.

Reply:
Sure, you can have a look at this `foundry.toml` - it has a lot of useful configurations: https://github.com/PaulRBerg/foundry-template/blob/main/foundry.toml.

As stated before, I'd suggest to allow the user to use as many networks as they want by removing the `$NETWORK` restriction from `.env`. It's also worth looking at the `.env.example` file within that repository and following that pattern.

On another note, everything else looks good to me. @Zizzamia

Reply:
okay makes sense! @gabrielstoica `bad fork_url provider: base_sepolia` is also happening because you are prob on an older version of foundry, let me know if that is not the case.

When it comes to allowing multiple chains at a time, yes I agree. That is something, we can def try doing. 

Maybe we can restructure your PR for that and support `base mainnet` and `base sepolia` at the same time.